extensions/cornerstone/src/initMeasurementService.ts

```diff
@@ -188,25 +189,26 @@ const initMeasurementService = (
   return csTools3DVer1MeasurementSource;
 };
 
-const connectToolsToMeasurementService = (servicesManager: AppTypes.ServicesManager) => {
+const connectToolsToMeasurementService = ({
+  commandsManager,
+  servicesManager,
+}: {
```

Comment:
Please just pass the props through to children calls, so that if this gets updated to need the extension manager at some point or some other new manager, it can just be included withotu further change.